app/src/main/res/layout-land/topic_info_fragment.xml

```diff
@@ -20,30 +20,31 @@
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:paddingTop="@dimen/padding_24dp"
-      android:paddingBottom="24dp">
+      android:paddingBottom="@dimen/padding_24dp">
 
       <TextView
         android:id="@+id/topic_name_text_view"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginStart="32dp"
-        android:layout_marginEnd="32dp"
+        android:layout_marginStart="72dp"
+        android:layout_marginEnd="72dp"
+        android:layout_marginTop="8dp"
         android:fontFamily="sans-serif"
         android:text="@{viewModel.topic.name}"
         android:textColor="@color/oppiaPrimaryText"
         android:textSize="24sp"
-        app:layout_constraintBottom_toBottomOf="@+id/topic_thumbnail_image_view"
         android:layout_marginBottom="8dp"
```

Comment:
Remove this bottom margin because the margin between topic_name and story_count is 8dp only which is already mentioned in `story_count_text_view`